widgets/nuxeo-card.html

```diff
@@ -44,104 +44,133 @@
   <template>
     <style>
       :host {
+        display: block;
         -webkit-box-sizing: border-box; /* Safari/Chrome, other WebKit */
         -moz-box-sizing: border-box;    /* Firefox, other Gecko */
         box-sizing: border-box;         /* Opera/IE 8+ */
+        background-color: white;
```

Comment:
I know that this is a backport, but how this behaves on other themes?

Reply:
Ok, it is overwritten by --nuxeo-card that is defined on WebUI themes.
Still, we are probably adding unnecessary information to it.